frame/sudo/src/lib.rs

```diff
@@ -274,20 +274,22 @@ pub mod pallet {
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
 		/// The `AccountId` of the sudo key.
-		pub key: T::AccountId,
+		pub key: Option<T::AccountId>,
```

Comment:
Presumably if they don't have an account ID to act as the sudo key, they would not use a `GenesisConfig`?

Reply:
Perhaps, but if `GenesisConfig` needs a default...